lib/beaker/host/unix/exec.rb

```diff
@@ -154,6 +154,9 @@ def mkdir_p dir
   # Recursively remove the path provided
   # @param [String] path The path to remove
   def rm_rf path
+    unless path.start_with?('"', '\'') && path.end_with?('"', '\'')
```

Comment:
This should do the trick. Probably it's enough to check if the string starts with quotes.